src/m365/aad/commands/user/user-hibp.ts

```diff
@@ -0,0 +1,100 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import Command from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import commands from '../../commands';
+import Utils from '../../../../Utils';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  userName: string;
+  apiKey: string;
+  domain?: string
+}
+
+class HIBPCommand extends Command {
+  public get name(): string {
+    return commands.USER_HIBP;
+  }
+
```

Comment:
Looking at https://haveibeenpwned.com/API/v3#SampleResponse, won't we get so much data in the output that it won't fit the terminal window in the text output? If that's the case, we should consider listing a few key properties that are displayed by default, like: https://github.com/pnp/cli-microsoft365/blob/9a77715c2965907e8bda48491b83e2ebbf99f721/src/m365/aad/commands/app/app-role-list.ts#L43-L45

Reply:
By default we get trimmed down response, to get this full response we need to pass `truncateResponse=false`. Should we add another flag to our command specification where user will have an option to pass this parameter?

Reply:
Which properties are available in the trimmed response?